### PR TITLE
[IMP] developer: rework section on debug mode

### PR DESCRIPTION
**Pull request by ged-odoo**

### CODE REVIEW

**Comment by ged-odoo** [comment]
this pr applies some comments made by @AntoineVDV in PR https://github.com/odoo/documentation/pull/1250 (but it was already merged). @odoo/doc-review

File: content/applications/general/developer_mode.rst [hunk 1]
```diff
@@ -33,10 +33,22 @@ Activate through a browser extension
...
    :alt: View of odoo’s debug icon in a chrome’s toolbar
 
+Activate through the command palette
+====================================
+
+The command palette tool has a command to activate the debug mode: open it with
+the keyboard shortcut `ctrl+k`, then type `debug`: a command will show up to
+activate the debug mode. 
+
+.. image:: command_palette.png
```
Comment (AntoineVDV) [inline content/applications/general/developer_mode.rst:43]: Please move the file inside a new folder (if it doesn't exist yet) `developer_mode/`. We try to progressively move toward [this file structure](https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#media-files) and will someday script a migration of all media files. Until then, let's already adopt the new way.

File: content/developer/reference/frontend/framework_overview.rst [hunk 2]
```diff
@@ -420,34 +420,8 @@ The `debug` mode current value can be read in the :ref:`environment<javascript/e
...
-- edit action
-- manage filters
-- edit the current view
-- see the `fields view get`
-- and much more.
+.. seealso::
+   - :ref:`Activate the debug mode <developer-mode>`
 
 Assets mode
 -----------
```
Comment (AntoineVDV) [inline content/developer/reference/frontend/framework_overview.rst:427]:
```suggestion

.. _javascript/debug_mode/assets:

Assets mode
-----------
```

File: content/developer/reference/frontend/framework_overview.rst [hunk 3]
```diff
@@ -459,7 +433,7 @@ are generated as well. This makes it useful to debug all kind of javascript code
 Tests mode
 ----------
```
Comment (AntoineVDV) [inline content/developer/reference/frontend/framework_overview.rst:434]:
```suggestion
.. _javascript/debug_mode/tests:

Tests mode
----------
```

**Review by AntoineVDV — COMMENTED** [review]
I added two refs in suggestions that would fit the existing tip of the Developer Mode page if it's rewritten as follows:
```rst
.. tip::
   Additional modes are available for developers: `?debug=assets` enables the :ref:`assets mode <javascript/debug_mode/assets>`, and `?debug=tests` enables the :ref:`tests mode <javascript/debug_mode/tests>`.
```

@robodoo delegate+

**Comment by ged-odoo** [comment]
@robodoo r+

**Pull request closed without merging**